docs/_quarto.yml

```diff
@@ -0,0 +1,95 @@
+project:
+   type: website
+
+metadata-files:
+  - reference/_sidebar.yml
+
+website:
+   title: "Inspect Flow"
+   description: "Workflow orchestration for Inspect AI"
+   bread-crumbs: true
+   page-navigation: true
```

Comment:
I'm not sure where it is configured, but the left and right navigation at the bottom of the pages is incorrect. On "Basic Examples" there is a right arrow navigation option for "Introduction" which goes to an early point on the page so shouldn't have a right arrow.